earth_enterprise/src/server/wsgi/serve/publish/publish_servlet.py

```diff
@@ -70,6 +70,8 @@ def DoGet(self, request, response):
       return
 
     try:
+      logger.info( "Incoming request to publish_servlet: ")
```

Comment:
Q: Do want to start logging requests as part of this issue?  How much logging does this produce?